scripts/rollup.config.js

```diff
@@ -60,13 +62,13 @@ module.exports = {
   input: 'src/index.js',
   plugins: [
     resolve({
-      jsnext: true,
-      main: true,
+      mainFields: ['main', 'jsnext'],
     }),
     commonjs({
       include: 'node_modules/**',
       sourceMap: true,
       namedExports: {
+        'node_modules/carbon-components/umd/index.js': ['settings'],
```

Comment:
@joshblack Was your finding that `carbon-components-react` is picking up `umd` directory of `carbon-components` instead of the `es` directory? Just thought it may have contributed to increased bundle size.

Reply:
@asudoh ah that might be it actually, the `node-resolve` plugin updated their fields config so I'm sure I just need to add `module` to the list to pick up `es`, will try real quick